src/main/java/core/basesyntax/types/EngineTypes.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.types;
+
+public enum EngineTypes {
+    Electric, Hybrid, Gasoline, Diesel
```

Comment:
for all your enums - let's make all the enum names singular (EngineTypes -> EngineType)
and make all the values uppercase (ELECTRIC, HYBRID etc)